### PR TITLE
Updated titles and modified some text for betterment.

### PR DESCRIPTION
**Pull request by isandesh1986**
# What will the user learn in your tutorial?

# What should the user already know when starting the tutorial?

### CODE REVIEW

File: cobigen/index.asciidoc [hunk 1]
```diff
@@ -1,8 +1,12 @@
-= Tutorial for Java Backend Generation with CobiGen
+= Generate your code with devonfw
```
Comment (maybeec) [inline cobigen/index.asciidoc:1]: What to generate out of what? Is it a Java stub application out of nothing or services out of open api?

File: create-your-own-tutorial/index.asciidoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= Create a tutorial with the Tutorial-Compiler
+= Create your own tutorial with the Tutorial-Compiler
```
Comment (maybeec) [inline create-your-own-tutorial/index.asciidoc:1]: Remove tutorial compiler. Nobody wanting to write a tutorial at first place needs to know about it.

**Review by maybeec — CHANGES_REQUESTED** [review]

File: cobigen/index.asciidoc (hunk 1 above)
Comment (isandesh1986) [reply cobigen/index.asciidoc]: Done

File: create-your-own-tutorial/index.asciidoc (hunk 2 above)
Comment (isandesh1986) [reply create-your-own-tutorial/index.asciidoc]: Done

File: devonfw-ide/index.asciidoc [hunk 3]
```diff
@@ -3,6 +3,13 @@
 devonfw offers complete IDE solution integrated with a lot​ of integrated tooling and initial settings​. 
 It massively speeds up development, reduces risks and helps deliver better results.
 
+## Prerequisites
+* User should have a prior experience with Eclipse or any ide.
```
Comment (GuentherJulian) [inline devonfw-ide/index.asciidoc:7]: IDE in upper case letters

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: devonfw-ide/index.asciidoc (hunk 3 above)
Comment (isandesh1986) [reply devonfw-ide/index.asciidoc]: Done

**Pull request merged**